h/api/models/__init__.py

```diff
@@ -13,6 +13,7 @@
 
 from h.api.models import elastic
 from h.api.models.annotation import Annotation
+from h.api.models.document import ConcurrentDocumentMetaCreateError
```

Comment:
This doesn't need to be "public". 

For that matter, neither do the "create_or_update_document_*" functions...